src/main/java/fi/metatavu/metaform/server/files/FileUploadServlet.java

```diff
@@ -38,6 +39,36 @@
   @Inject
   private FileController fileController;
   
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+    String fileRef = req.getParameter("fileRef");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal "fileRef" 3 times. ([squid:S1192](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=57&issues=AWWZzNGaAJjEPbGfspZv&open=AWWZzNGaAJjEPbGfspZv)

[ISSUE_KEY]: AWWZzNGaAJjEPbGfspZv